_posts/2020-02-07-metal3-dev-env-install-deep-dive.md

```diff
@@ -0,0 +1,921 @@
+
+> ```sh
+> + sudo podman run -d --net host --privileged --name vbmc --pod infra-pod -v /opt/metal3-dev-env/virtualbmc/vbmc:/root/> .vbmc -v /root/.ssh:/root/ssh quay.io/metal3-io/vbmc
+> ```
+<br>
+<br>
+> * **The sushy-tools container.** 
+<br>
+<br>
+> This container mounts the `/opt/metal3-dev-env/virtualbmc/sushy-tools config folder and the `/root/.ssh` local folder as well. The functionality is similar as the `vbmc`, however this use redfish instead of ipmi to connect to the BMC.
+
+>```sh
+> + sudo podman run -d --net host --privileged --name sushy-tools --pod infra-pod -v /opt/metal3-dev-env/virtualbmc/> sushy-tools:/root/sushy -v /root/.ssh:/root/ssh quay.io/metal3-io/sushy-tools
+> ```
+
+<br>
+
+> **Information:** At this point the virtual infrastructure must be ready to apply the Kubernetes specific configuration. Note that all VMs specified by `NUMBER_NODES` and `minikube` must be shut down and the defined virtual network must be active:
+
+
+```sh
+[alosadag@smc-master metal3-dev-env]$ sudo virsh list --all
+ Id    Name                           State
+----------------------------------------------------
+ -     minikube                       shut off
+ -     node_0                         shut off
+ -     node_1                         shut off
+ -     node_2                         shut off
+
+[alosadag@smc-master metal3-dev-env]$ sudo virsh net-list --all
+ Name                 State      Autostart     Persistent
+----------------------------------------------------------
+ baremetal            active     yes           yes
+ default              active     yes           yes
+ minikube-net         active     yes           yes
+ provisioning         active     yes           yes
+```
+
+<iframe width="1110" height="625" style="height: 625px" src="https://www.youtube.com/embed/HfPxDqC2sH8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<br>
+
+## **Step 3: Launch the management cluster (minikube)**
+
+The third script called `03_launch_mgmt_cluster.sh` basically configures `minikube` to become a Metal³ management cluster. On top of minikube the `baremetal-operator`, `capi-controller-manager`, `capbm-controller-manager` and `cabpk-controller-manager` are installed in the metal3 namespace.
+
+In a more detailed way, the script clones the `BareMetal operator` ([BMO](https://github.com/metal3-io/baremetal-operator)) and `Cluster API Provider for Managed Bare Metal Hardware operator` ([CAPBM](https://github.com/metal3-io/cluster-api-provider-baremetal)) git repositories, creates the cloud.yaml file and starts the `minikube` virtual machine. Once `minikube` is up and running, the `BMO` is built and executed in minikube's Kubernetes cluster. 
```

Comment:
Ah, here's the link to the BMO repo!

Maybe add it earlier, when it's first mentioned

Reply:
Maybe briefly mention what this `cloud.yaml` file is about